### ISSUE TITLE
Safari 14 supports instantiatable EventTarget

### ISSUE THREAD

**Issue by eligrey** [mdn/browser-compat-data#7019]
[The browser compat table for EventTarget()](https://wiki.developer.mozilla.org/en-US/docs/Web/API/EventTarget/EventTarget#Browser_compatibility) should indicate that this is supported by Safari 14. It may also be supported by earlier versions of Safari, but I only tested Safari 14.

**Comment by sideshowbarker** [comment]
Thanks for reporting this. Any update was already merged in #6984 — 

https://github.com/mdn/browser-compat-data/blob/7ae2f86e02f9def5beabd8264b027769e6c885c8/api/EventTarget.json#L52-L86

…but it seems that MDN article hasn’t picked that up yet. I don’t know myself how long it typically takes for changes made here to get propagated to MDN. Maybe @ddbeck knows

**Comment by ddbeck** [comment]
> I don’t know myself how long it typically takes for changes made here to get propagated to MDN. Maybe @ddbeck knows

Typically, changes work their way into MDN in a few days, but in some cases it takes up to a month. I did not do a BCD release last week, so this change has has taken longer than usual.

For a change in BCD to be reflected in an MDN page, this is what needs to happen:

1. The PR is merged.
2. BCD is released. This usually happens on Thursdays. I didn't manage to do a release last week, so #6984 hasn't shipped yet.
3. The BCD release is deployed to MDN. This usually happens the same day as releases.
4. The page on MDN needs to be regenerated. This can happen when:

   - The page is edited
   - The page is regenerated manually with shift+refresh on `wiki.developer.mozilla.org` (i.e., you click the edit button on a page, then force refresh it).
   - The MDN dev team force refreshes _all_ pages with BCD tables. This usually happens monthly.

Long story short, this data ought to be deployed to MDN on Thursday, October 29; I'll plan to refresh the page after that. Thanks for your patience on this one.

**Comment by ddbeck** [comment]
The [`EventTarget()` page](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/EventTarget#Browser_compatibility) now reflects the latest data (as of v2.0.4). I'm going to close this, but do comment if there are any remaining issues. Thank you!
